manifests/cli/params.pp

```diff
@@ -7,6 +7,7 @@
   $password = undef
   $refresh_cache = false
   $request_timeout = 120
+  $use_sessions = false
```

Comment:
Is false the built in default? I'd prefer to follow the hammer defaults.

Reply:
According to the default `foreman.yml` provided on Debian, `false` is the default.